.github/workflows/test-fast.yml

```diff
@@ -73,6 +73,10 @@ jobs:
     runs-on: ${{ matrix.platform }}
 
     steps:
+      - name: Reduce TESTPARALLELISM on Windows to work around OOM
+        if: ${{ matrix.platform == 'windows-latest' }}
+        run: |
+          echo "TESTPARALLELISM=1" >> $GITHUB_ENV
```

Comment:
Note for self on this GitHub-ism: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-environment-variable